.github/workflows/deploy-staging-and-prepare-release.yaml

```
@@ -39,6 +39,9 @@ jobs:
     needs: prepare-release
     if: always()
     runs-on: ubuntu-22.04
+    permissions:
+      id-token: write # needed by aws-actions/configure-aws-credentials
+      contents: read
```

Missing `checks: write`

```suggestion
      contents: read
      checks: write
```

Why: 2 steps in this job are calling [this composite action](https://github.com/saleor/saleor-dashboard/blob/20171a40e05dd84aef0cceebb94b707112fdd7d0/.github/actions/checks/update-pull-request-check/action.yml#L63-L69):
1. https://github.com/saleor/saleor-dashboard/blob/439481f9677ea7cccdf0491a7e3f5a6d534b8758/.github/workflows/deploy-staging-and-prepare-release.yaml#L99-L102
2. https://github.com/saleor/saleor-dashboard/blob/439481f9677ea7cccdf0491a7e3f5a6d534b8758/.github/workflows/deploy-staging-and-prepare-release.yaml#L140-L142